datahub-web-react/src/components/entity/dataset/DatasetPage.tsx

```diff
@@ -71,7 +71,7 @@ export const DatasetPage: React.VFC = () => {
                 path: TabType.Ownership.toLowerCase(),
                 content: (
                     <OwnershipView
-                        owners={(ownership && ownership.owners) || []}
+                        initialOwners={(ownership && ownership.owners) || []}
```

Comment:
Should we use EMPTY_LIST here instead of []?

Reply:
sure, good point